This commit adds the pdf.js library

little9 opened the pull request:
…  and the viewer from Firefox to to the application in the `/public` directory. The viewer needs to be hosted statically.

This also adds a 'View Document' button to the show page that links to the portion of the page where the viewer is displayed.

Closes #14

no-reply commented:
Are the views/partials included in this PR copied in from Hyrax? 

I think it would help in future upgrades for the commit history to reflect the copy-in separate from the changes specific to this feature. Should we adopt a practice of including a commit that copies views and partials in exactly (maybe with a note about which commit/tag of Hyrax they are copied from), separate form any changes we may actually want to make?

little9 committed:
Add pdf.js viewer assets from:
https://github.com/mozilla/pdf.js/tree/gh-pages

These assets need to be hosted statically, as the viwer
relies on an `<iframe>` including the html.

little9 committed:
This commit alters default Hyrax views to include the pdf.js viewer in the UI.

It adds a 'View Document' button to the show page that links to the portion of the page where the viewer is displayed.

little9 committed:
Copy in show.html.erb view and _pdf.html.erb partials from
Hyrax 2.0.0:

https://github.com/samvera/hyrax/blob/v2.0.0/app/views/hyrax/base/show.html.erb
https://raw.githubusercontent.com/samvera/hyrax/v2.0.0/app/views/hyrax/file_sets/media_display/_pdf.html.erb

little9 commented:
@no-reply I went back and added a commit that includes links to the sources for the overrides.
